### PR TITLE
Merge rc 1 6 into sovereign 22 jun 2023

### PR DESCRIPTION
**Pull request by mariusmihaic**
Merge rc 1 6 into sovereign 22 jun 2023

### CODE REVIEW

File: genesis/process/disabled/disabled_test.go [hunk 1]
```diff
@@ -271,12 +272,12 @@ func TestScheduledTxsExecutionHandler(t *testing.T) {
 	require.False(t, handler.IsInterfaceNil())
 
 	hash, err := handler.GetScheduledRootHashForHeader([]byte{})
-	require.Equal(t, make([]byte, 0), hash)
-	require.Nil(t, err)
```
Comment (iulianpascalau) [inline genesis/process/disabled/disabled_test.go]: why this change? also on L280

Comment (mariusmihaic) [reply genesis/process/disabled/disabled_test.go]: Because the stub was changed to return `nil, err` instead of `make(), nil`

**Review by iulianpascalau — APPROVED** [review]

**Review by raduchis — APPROVED** [review]
